py/desispec/quicklook/qlconfig.py

```diff
@@ -278,17 +290,25 @@ def io_qa(self,qaname):
         """
         Specify the filenames: yaml and png for the given qa output
         """
-        filemap={'Bias_From_Overscan': 'ql_getbias',
-                 'Get_RMS' : 'ql_getrms',
-                 'Count_Pixels': 'ql_countpix',
-                 'Calc_XWSigma': 'ql_xwsigma',
-                 'CountSpectralBins': 'ql_countbins',
-                 'Sky_Continuum': 'ql_skycont',
-                 'Sky_Peaks': 'ql_skypeak',
-                 'Sky_Residual': 'ql_skyresid',
-                 'Integrate_Spec': 'ql_integ',
-                 'Calculate_SNR': 'ql_snr'
-                 }
+        if self.conf["Flavor"] == 'arcs':
+            filemap={'Bias_From_Overscan': 'ql_getbias_arc',
+                     'Get_RMS' : 'ql_getrms_arc',
```

Comment:
Same here. Do we explicitly need "arc" string in the file names? Isn't unique expid sufficient. It seems there are many of these.